lib/galaxy/util/__init__.py

```diff
@@ -721,7 +721,7 @@ def __init__(self, params, sanitize=True):
                     key not in self.NEVER_SANITIZE and
                     True not in [key.endswith("|%s" % nonsanitize_parameter) for
                                  nonsanitize_parameter in self.NEVER_SANITIZE]):
```

Comment:
I'm going to leave this as is given the disagreement and since this PR doesn't actually touch those lines anyway - is that okay with everyone for now? @nsoranzo if you want to open a followup PR that would be the right place to discuss this further I think?